.changeset/clean-tables-crash.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/flight-icons": minor
```

Comment:
Question: do you need two separate change logs?

Reply:
@MelSumner ysing the `yarn changelog` command, it was not clear (to me) how to apply different semver type to the packages, so I kept it simple and run it twice (and this created two files). I don't think this should make a difference for `changelog` (@Dhaulagiri can you confirm this?)

Reply:
It should be possible to combine them via `yarn changelog` but in practice I don't think it'll make a difference: changesets will be able to figure it out similar to if these were part of completely separate changes.